Zend/zend.c

```diff
@@ -5,12 +5,7 @@
    | Copyright (c) Zend Technologies Ltd. (http://www.zend.com)           |
    +----------------------------------------------------------------------+
    | This source file is subject to version 2.00 of the Zend license,     |
```

Comment:
Ugh, LICENSE SAYS: `The PHP License, version 3.01`

Reply:
It refers to the `LICENSE` file of the same directory: https://github.com/php/php-src/blob/master/Zend/LICENSE

Reply:
Ah, thanks!